src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/overlays/LinkTypeOverlay.js

```diff
@@ -11,9 +11,11 @@ import type {LinkTypeOverlayProps} from '../types';
 export default class LinkTypeOverlay extends React.Component<LinkTypeOverlayProps> {
     render() {
         const {
+            query,
```

Comment:
i think it would be nice to support these properties in the `link` content type similar to the anchor implementation: https://github.com/sulu/sulu/blob/b45f91ff36b468d200b254f894368cb0c3380671/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Link.js#L19-L21

but its okay for me to do that in another pull request